src/pages/content-edit/lib/delegator-validity.js

```diff
@@ -0,0 +1,147 @@
+import m from "mithril";
+import debounce from "lodash.debounce";
+
+import css from "../invalid-msg.css";
+
+function reqPrefix(name) {
+    return "Required: " + name;
+}
+
+export default function Validity(content) {
+    var v = this;
```

Comment:
Use `self` as an alias for `this`. `ctrl` is also an ok alias within a mithril controller.